### PR TITLE
Make `Label3D` obey auto translation

### PR DESCRIPTION
**Pull request by YeldhamDev**
A note was also added to `TextMesh`'s doc to explain why the same can't be done to it.

### CODE REVIEW

File: doc/classes/TextMesh.xml [hunk 1]
```diff
@@ -53,6 +53,7 @@
 		</member>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
 			The text to generate mesh from.
+			[b]Note:[/b] Due to being a [Resource], it doesn't follow the rules of [member Node.auto_translate_mode]. If turning off translation is desired, it should be done manually with [method Object.set_message_translation].
```
Comment (AThousandShips) [inline doc/classes/TextMesh.xml:56]:
```suggestion
			[b]Note:[/b] Due to being a [Resource], it doesn't follow the rules of [member Node.auto_translate_mode]. If disabling translation is desired, it should be done manually with [method Object.set_message_translation].
```

**Review by AThousandShips — APPROVED** [review]
LGTM

File: doc/classes/TextMesh.xml (hunk 1 above)
Comment (YeldhamDev) [reply doc/classes/TextMesh.xml]: Done.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**